german/GUI_Automation/src/test/java/at05ui/steps/SendDisplayedInbox.java

```diff
@@ -0,0 +1,32 @@
+package at05ui.steps;
+
+import at05ui.sampleapp.Entities.Mail;
+import at05ui.sampleapp.ui.InboxPage;
+import at05ui.sampleapp.ui.LoginPage;
+import at05ui.sampleapp.ui.NextPage;
+import at05ui.sampleapp.ui.PageTransporter;
+import cucumber.api.java.en.Then;
+
+/**
+ * Created by Administrator on 11/22/2017.
+ */
+public class SendDisplayedInbox {
+  LoginPage loginPage;
+  InboxPage inboxPage;
+  NextPage nextPage;
+  Mail mail;
+
+  public SendDisplayedInbox(Mail mail) {
+    this.mail = mail;
+  }
+
+  @Then("^the email send should be displayed in the Inbox list$")
+  public void displayedInbox() throws Throwable {
+    //inboxPage.searchMail();
+    //inboxPage.deleteMailBySubject("AT05");
+    //Thread.sleep(3000);
+    inboxPage = PageTransporter.getInstance().navigateToInboxPage();
```

Comment:
You can declare a PageTransporter pageTransporter; as global variable, and in the constructor of the class you can do
pageTransporter = PageTransporter.getInstance(); then you can use this variable in any step.

Reply:
I only use it once, it would not be necessary to add an additional variable